runtime/runtime_test.go

```diff
@@ -2337,48 +2350,6 @@ func TestRuntimeTransaction_AddPublicKey(t *testing.T) {
 	}
 }
 
-func TestRuntimeCyclicImport(t *testing.T) {
```

Comment:
Is this moved somewhere? Or covered by another test? `TestInterpretImportMultipleProgramsFromLocation` maybe?

Reply:
This PR initially removed an explicit check for cyclic imports. Cyclic imports where still not supported, but instead of getting an dedicated error for cyclic imports, the checker would still fail, but only through implicit "not declared" errors.

Thinking about it a bit more, this is not very user-friendly: a developer might only see a "not declared" error, see they have a valid declaration, but would not realize the actual cause is a cyclic import. 

I've added the explicit check and error back in [`2b713fc` (#333)](https://github.com/onflow/cadence/pull/333/commits/2b713fcff712cbb54271811a50f9e15c1828503a).

Regarding this test case in particular: This is a runtime check, but all it really tests is that the checker detects the cyclic import, so it shouldn't be a runtime test in the first place, but rather a checker test.

I've improved https://github.com/onflow/cadence/pull/333/files#diff-4fb398062392664e4c2200a9025f9f42R543 and also added another test for a mutually recursive import here:
https://github.com/onflow/cadence/pull/333/files#diff-4fb398062392664e4c2200a9025f9f42R586.
Basically, these two checker tests cover this test and we don't need the runtime test here.